extensions/homeassistant/src/components/hooks.tsx

```diff
@@ -41,21 +42,13 @@ export function useHAStates(): {
   error?: Error;
   isLoading: boolean;
 } {
-  const [states, setStates] = useState<State[]>();
+  const [states, setStates] = useCachedState<State[]>("states");
```

Comment:
I suggest to use a cache for storing states. This way, when we execute a command like "Lights," the list won't be empty until we receive the first state update. Instead, it should already be populated with the entries from the last time we ran any command.

Reply:
Works nice. A little snappier